tests/test_utils.py

```diff
@@ -20,3 +20,9 @@ def test_cannot_get_pv_prefix():
 
 def test_centred_pixel():
     assert centred_pixel(50, 3) == [47, 48, 49, 50, 51, 52, 53]
+
+
+@pytest.mark.parametrize("mpl_backend", ["qt", "something_else"])
```

Comment:
should we test with `qt4Agg`, `qt5Agg`, `qtCairo` here rather than just `qt` to test that we match them all?